api-spec/extensions/sort/README.md

```diff
@@ -2,27 +2,44 @@
+By default, the STAC search endpoint `/search` returns results in no specified order.  Whatever order the results are in 
+is up to the implementor, and will typically default to an arbitrary order that is fastest for the underlying data store 
+to retrieve results.
  
- The Sort API Extension adds a new parameter, `sortby`, that allows the user to define fields by which to sort results. Only string, numeric, and datetime attributes of Item (`id` and `collection` only) or Item Properties (any attributes) may be used to sort results.  It is not required that implementations support sorting over all attributes, but implementations should return an error when attempting to sort over a field that does not support sorting. 
+The Sort API Extension adds a new parameter, `sortby`, that allows the user to define fields by which to sort results. 
+Only string, numeric, and datetime attributes of Item (`id` and `collection` only) or Item Properties (any attributes) 
+may be used to sort results.  It is not required that implementations support sorting over all attributes, but 
+implementations should return an error when attempting to sort over a field that does not support sorting. 
 
-Two values for direction are supported: "asc" (ascending) or "desc" (descending). If the direction is not specified, the value is ascending. The `sortby` value is an array, so multiple sort fields can be defined which will be used to sort the data in the order provided (e.g., first by `datetime`, then by `eo:cloud_cover`).
+Fields may be sorted in ascending or descending order.  The syntax between GET requests and POST requests with a JSON 
+body vary.  The `sortby` value is an array, so multiple sort fields can be defined which will be used to sort 
+the data in the order provided (e.g., first by `datetime`, then by `eo:cloud_cover`).
 
-## GET or POST Form
+## HTTP GET (or POST Form)
 
-When calling `/search` using GET or POST with `Content-Type: application/x-www-form-urlencoded` or `Content-Type: multipart/form-data`, the semantics are the same, except the syntax is a single parameter `sortby` with a comma-separated list of "<name>|<direction>" definitions.  It is recommended that in implementations, direction be mandatory, and that an error should result from not specifying a direction.
+When calling `/search` using GET (or POST with `Content-Type: application/x-www-form-urlencoded` or 
+`Content-Type: multipart/form-data)`, a single parameter `sortby` with a comma-separated list of item field names should 
+be provided. The field names may be prefixed with either "+" for ascending, or "-" for descending.  If no sign is 
+provided before the field name, it will be assumed to be "+". 
 
 Examples of `sortby` parameter:
 
-    GET /search?sortby=created|asc
+    1) GET /search?sortby=properties.created
```

Comment:
Semantically, it would be better to use

1. `GET ...`
2. `GET ...`

Reply:
Ok.  That was the existing syntax in the document, but I will change it.